src/store/sliceGoods/sliceAllGoods.ts

```diff
@@ -0,0 +1,51 @@
+import {createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { LOAD_STATUSES } from '../../constants'
+import {Good,Api } from "api/api";
```

Comment:
сделай рекмпорт через api/index, чтобы тут стало
```suggestion
import {Good,Api } from "api";
```